client/extensions/woocommerce/app/order/create.js

```diff
@@ -131,18 +155,35 @@ export default connect(
 		const siteId = site ? site.ID : false;
 		const orderId = getCurrentlyEditingOrderId( state );
 		const isSaving = isOrderUpdating( state, orderId );
-		const hasOrderEdits = ! isEmpty( getOrderEdits( state ) );
+		const orderEdits = getOrderEdits( state );
+		const hasOrderEdits = ! isEmpty( omit( orderEdits, [ 'currency' ] ) );
 		const order = getOrderWithEdits( state );
+		const settingsGeneralLoaded = areSettingsGeneralLoaded( state, siteId );
+		const currencySettings = getPaymentCurrencySettings( state, siteId );
+		const currencyCode = currencySettings.value || 'USD';
 
 		return {
+			currencyCode,
 			hasOrderEdits,
 			isSaving,
 			order,
+			orderEdits,
 			orderId,
+			settingsGeneralLoaded,
 			site,
 			siteId,
 		};
 	},
 	dispatch =>
-		bindActionCreators( { clearOrderEdits, editOrder, saveOrder, sendOrderInvoice }, dispatch )
+		bindActionCreators(
+			{
+				clearOrderEdits,
+				editOrder,
+				fetchSettingsGeneral,
+				getPaymentCurrencySettings,
```

Comment:
This is a selector, not an action, right?

Reply:
Oops! Fixed in https://github.com/Automattic/wp-calypso/pull/23568/commits/8a7c403776596e9294f02baf3bf29b7b5c04d140